packages/driver/src/cy/commands/navigation.ts

```diff
@@ -1162,7 +1194,8 @@ export default (Commands, Cypress, cy, state, config) => {
         // so that we nuke the previous state. subsequent
         // visits will not navigate to about:blank so that
         // our history entries are intact
-        if (!hasVisitedAboutBlank) {
+        // TODO: is this okay to skip for multi-domain?
+        if (!hasVisitedAboutBlank && !Cypress.state('isMultiDomain')) {
```

Comment:
What's the reason for skipping this for multi-domain?

Reply:
It felt odd to navigate the AUT to `about:blank` during the middle of a test when that wouldn't happen during a normal workflow.

Reply:
Ah I see. I think we still need to do this if the first visit in the test is in the switchToDomain callback, right? Otherwise, it won't visit about:blank before visiting. We might need to sync the `hasVisitedAboutBlank` between the primary and the spec bridge.

Reply:
After discussion, I am going to add support for `about:blank` in multi-domain. While we anticipate to turn on `experimentalSessionSupport` at the same time of `experimentalMultiDomain` we do not want to be tied to this decision.

Reply:
Added `about:blank` support here. [5aafb8f](https://github.com/cypress-io/cypress/pull/20467/commits/5aafb8f3a723b6ddea1655fcc032e856ef1737d4)